cpa/tester/runner.go

```diff
@@ -176,21 +176,28 @@ func (runner *Runner) runTest(policy *cpa.Policy, results chan<- Result, t Named
 		return internal.Merge(parent.Meta, t.Meta)
 	}()
 
+	decision := func() any {
+		if t.Decision == nil {
+			return parent.Decision
+		}
+		return internal.Merge(parent.Decision, t.Decision)
+	}()
+
 	name := t.Name
 	if parent.Name != "" {
 		name = parent.Name + "/" + name
 	}
 
-	if runner.include == nil || runner.include.MatchString(name) {
+	if decision != nil && (runner.include == nil || runner.include.MatchString(name)) {
 		eval, _ := policy.Eval(context.Background(), "data", input, cpa.Meta(meta))
 
 		start := time.Now()
-		var decision any = internal.Must(policy.Decide(context.Background(), input, cpa.Meta(meta)))
+		var actualDecision any = internal.Must(policy.Decide(context.Background(), input, cpa.Meta(meta)))
```

Comment:
Nit:
Any specific reason why we explicitly declare the variable here?

Reply:
because afterwards i reassign the same variable to a raw `any` using json marshalling and unmarshalling, so that it can be compared against the user's decision with equal types!